RallyCat.Core/Services/RallyService.cs

```diff
@@ -64,8 +69,9 @@ public QueryResult GetRallyItemByQuery(RallySlackMapping map, List<string> reque
 
         public List<dynamic> GetKanban(RallySlackMapping map)
         {
-            
-            RallyRestApi restApi = _pool.GetApi(map.UserName, map.Password);
+            var rallyToken = "_LrqfvZJdQ8O2UtDHia4uxpNpFBaZUFcoVuEWLpxJy0";
```

Comment:
Hard-coded value should be moved to config.json, please see
https://github.com/daily2432121/RallyCat/blob/master/RallyCat.WebApi/config.json.template

Additional changes need to be added to parse them